meta-mender-core/recipes-bsp/grub-mender-grubenv/grub-mender-grubenv_1.1.0.bb

```diff
@@ -41,6 +41,25 @@ python() {
         bb.build.addtask('do_provide_debug_log', 'do_patch', 'do_unpack', d)
 }
 
+python do_partuuid_error_check() {
+        if bb.utils.contains('DISTRO_FEATURES', 'mender-image-gpt', True, False, d) or bb.utils.contains('DISTRO_FEATURES', 'mender-image-uefi', True, False, d):
+            test_part_uuid(d, 'MENDER_ROOTFS_PART_A_UUID')
+            test_part_uuid(d, 'MENDER_ROOTFS_PART_B_UUID')
+            test_part_uuid(d, 'MENDER_DATA_PART_UUID')
+            test_part_uuid(d, 'MENDER_BOOT_PART_UUID')
+        elif bb.utils.contains('DISTRO_FEATURES', 'mender-image-sd', True, False, d) or bb.utils.contains('DISTRO_FEATURES', 'mender-image-bios', True, False, d):
+            test_disk_identifier(d, 'MENDER_DISK_IDENTIFIER')
```

Comment:
* I would make it a new `if` instead of `elif`. Although we are not testing it, theoretically `DISTRO_FEATURES` could contain both cases. Might as well catch both errors then.
* Indentation should be four spaces.

Reply:
TBH, I'm not sure either. I'm just thinking there is no reason why we couldn't have both, just in case.